<launch>
	<node name="takktile" 
	pkg="takktile_ros" 
	type="takktile_node.py" />


	<node pkg="imu_files" 
	name="collect_data2" 
	type="collect_data2.py" /> 
	
	<node pkg="imu_files" 
	name="real_time" 
	type="real_time.py" 
	output = "screen"/> 

</launch>


